test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t

```diff
@@ -0,0 +1,59 @@
+  $ dune runtest dune-file | sed "s/(lang dune .*)/(lang dune <version>)/" | dune_cmd sanitize
```

Comment:
How does this test work?

Reply:
I've added a description to this test if that's what you meant.

Reply:
Thanks. Yeah, I was trying to understand what this was testing.

Reply:
While we are at it, I made the test a little bit more focused.